test/ca/mcgill/cs/stg/jetuml/commands/TestDeleteNodeCommand.java

```diff
@@ -0,0 +1,79 @@
+package ca.mcgill.cs.stg.jetuml.commands;
+
+import static org.junit.Assert.*;
+
+import java.awt.geom.Point2D;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ca.mcgill.cs.stg.jetuml.diagrams.ClassDiagramGraph;
+import ca.mcgill.cs.stg.jetuml.graph.ClassNode;
+import ca.mcgill.cs.stg.jetuml.graph.Graph;
+import ca.mcgill.cs.stg.jetuml.graph.Node;
+
+public class TestDeleteNodeCommand {
+    private Graph aGraph;
+    private Field aNeedsLayout;
+    private Field aNodesToBeRemoved;
+    private Node aNode;
+    private ArrayList<Node> aRootNodes;
+    private DeleteNodeCommand aDeleteNodeCommand;
+    private boolean hasExecuted = false;
```

Comment:
`hasExecuted` should be named with the usual convention for fields. More importantly, I don't get how this field is supposed to be set as a result of the command being executed.